process/transactionLog/printTxLogProcessor.go

```diff
@@ -0,0 +1,64 @@
+package transactionLog
```

Comment:
you do not have code coverage for this file. Since it is only used in integration tests maybe consider moving it in `testscommon` package.